tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -292,6 +292,7 @@ private static void GenerateMainProject(ApiMetadata api, string directory, HashS
                 // Build-related properties
                 new XElement("Version", api.Version), // TODO: Version, or VersionPrefix/VersionSuffix?
                 new XElement("TargetFrameworks", targetFrameworks),
+                new XElement("LangVersion", "latest"),
```

Comment:
Well I guess technically we had that situation already, right? We were implicitly opted into the latest major version, which arguably has more of a chance for breaking changes, right? I assume minor versions would be safer in that regard. But if you feel otherwise, I can make it explicit.